packages/swap/prisma/schema.prisma

```diff
@@ -417,23 +418,19 @@ model StateChainError {
   @@schema("public")
 }
 
-model QuoteResult {
-  id                 Int           @id @default(autoincrement())
-  srcAsset           InternalAsset
-  destAsset          InternalAsset
-  depositAmount      Decimal       @db.Decimal(30, 0)
-  quoterIntermediate Decimal?      @db.Decimal(30, 0)
-  quoterOutput       Decimal       @db.Decimal(30, 0)
-  quoterDuration     Int
-  quoterFees         Json?
-  poolIntermediate   Decimal?      @db.Decimal(30, 0)
-  poolOutput         Decimal       @db.Decimal(30, 0)
-  poolDuration       Int?
-  poolFees           Json?
-  createdAt          DateTime      @default(now())
-  version            Int
-
-  @@index([srcAsset, destAsset])
+model Quote {
+  id                         Int                @id @default(autoincrement())
+  swapDepositChannel         SwapDepositChannel @relation(fields: [swapDepositChannelId], references: [id])
+  swapDepositChannelId       BigInt             @unique
+  srcAsset                   InternalAsset
+  destAsset                  InternalAsset
+  depositAmount              Decimal            @db.Decimal(30, 0)
+  intermediateAmount         Decimal?           @db.Decimal(30, 0)
+  egressAmount               Decimal            @db.Decimal(30, 0)
+  estimatedPrice             Decimal
+  executedPrice              Decimal?
```

Comment:
will set this in an upcoming pr

Reply:
what happens if a channel has been reused? we'd always store the latest executedPrice?

Reply:
im think ill store only the first one for now. if its already set, i wont overwrite it

Reply:
I think we can just store the quote here and a link to the deposit channel already enables us to find the executed swaps. then we can generate the report that we want based on the 3 tables (quote, swapDepositChannel and swapRequest)

Reply:
yeah thats true, but its less convenient. dont have a strong opinion though - im also fine with keeping this table to a minimum